### ISSUE TITLE
Windows 7 does not support virtual desktop

### ISSUE THREAD

**Issue by JanPokorny** [joelverhagen/TorSharp#58]
Attempting to use virtual desktop (which is the default option on Windows) throws an exception on Windows 7. This is easily resolved by checking `System.Environment.OSVersion.Version` to be less than 6.2 ([details](https://stackoverflow.com/questions/2819934/detect-windows-version-in-net)) and reverting to standard mode. You may consider implementing this check in the library itself.

**Comment by JanPokorny** [comment]
The current workaround for supporting Windows 7 is having this in `TorSharpSettings`:

```csharp
ToolRunnerType =
    Environment.OSVersion.Version < new Version(6, 2)
    ? ToolRunnerType.Simple
    : ToolRunnerType.VirtualDesktop
```

**Comment by joelverhagen** [comment]
I'd like to get this built into TorSharp, I'm just a bit slammed with other commitments right now.

**Comment by joelverhagen** [comment]
This has been fixed in 2.5.0. Thanks for the suggestion!
https://www.nuget.org/packages/Knapcode.TorSharp/2.5.0
